_pages/response-guide.html

```diff
@@ -0,0 +1,123 @@
+        <li>Do not share any of the information with anyone else.</li>
+      </ul>
+
+      <p>In speaking to a reporter, be cautious in your wording. Try to be understanding and compassionate, and make them feel heard and taken seriously. However, do not make commitments on whether the conference will see this as a violation, or what action will be taken.</p>
+      <p>In urgent cases, where it is not possible to contact a CARE team member and you believe immediate action is essential, you may intervene unilaterally. For example, you may consider this:</p>
+
+      <ul>
+        <li>When any delay of action will likely bring unacceptable further harm to others. Ongoing aggression or serious harassment are examples of this.</li>
+        <li>If you feel people are in immediate physical danger, it can be appropriate to contact local law enforcement.</li>
+        <li>If you witness a presentation which is clearly in violation of the Code of Conduct repeatedly or very seriously. You might simply say “I’m sorry, this presentation cannot be continued at the present time”. Examples of when this is justified could be significant threats of violence, harassment of others, or continuous sexist jokes. Do not end a presentation in cases like a few inappropriate jokes - in that case do report it to the CARE team.</li>
+      </ul>
+
+      <p>When possible, even in these cases, consult with another organiser or volunteer if you see one nearby.</p>
+    </div>
+  </section>
+
+  <section class="section-pad">
+    <div class="row column">
+      <h2 id="coc-response-team">CoC response team</h2>
+      <p>The Code of Conduct Active Response Ensurers (CARE) are in charge of handling reports until their completion. It is their task to ensure all reports are handled timely and professionally. Our CARE team is still in the making - volunteers are welcome.</p>
+      <p>Participants look towards the CARE team for appropriate behaviour, even more than other staff, and that makes it essential for their behaviour to remain within the Code of Conduct. Violations by CARE team members can severely erode trust in the Code of Conduct process.</p>
+      <p>CARE team members almost never act on their own, including tasks like replying to Code of Conduct related e-mails. If it’s not possible to meet with the entire CARE team in time, at least one other team member should be consulted.</p>
+
+      <hr>
+
+      <h3 id="receiving-a-report">Receiving a report</h3>
+      <p>Reports are typically received by e-mail, in&nbsp;Discord, or from another staff member. If the report was received from someone else, it may be best to meet with the original reporter first, depending on how complete the information is. The information the CARE team aims to collect is listed in the Code of Conduct reporting guide, but sometimes not all information is available.</p>
```

Comment:
```suggestion
      <p>Reports are typically received by e-mail, in Discord, or from another staff member. If the report was received from someone else, it may be best to meet with the original reporter first, depending on how complete the information is. The information the CARE team aims to collect is listed in the Code of Conduct reporting guide, but sometimes not all information is available.</p>
```